resources/scripts.js

```diff
@@ -173,10 +173,10 @@ function playerLose() {
     betButtonState(true);
 }
 
-function placeBet() {
+function makeBet() {
     betAmount = betDialogElement.value;
```

Comment:
For Event Listeners, is it possible to pass another argument? For example, instead of creating a variable "betAmount", could I simply pass betDialogElement.value as an argument and use it throughout the function?

Reply:
You could or just search for the `betDialogElement` using a query selector to get the value within the function itself